Views/home.stencil

```diff
@@ -0,0 +1,97 @@
+<!--
```

Comment:
I think older versions of Internet Explorer won't render this - they require the file to start with the `<!DOCTYPE HTML>`. I think just remove this header...

Reply:
Removed